### ISSUE TITLE
There isn't cjdns in openwrt 15.05.1

### ISSUE THREAD

**Issue by whoizit** [SeattleMeshnet/meshbox#48]
Only luci-app-cjdns. Should I do downgrade to 15.05?

**Comment by dvn0** [comment]
For some reason it was not included upstream. I don't know why. So, you can try building it yourself for 15.05.1

**Comment by gema-arta** [comment]
https://www.mail-archive.com/openwrt-devel@lists.openwrt.org/msg37072.html

```
>    ERROR: package/feeds/routing/cjdns failed to build.
```

**Comment by ghost** [comment]
Yeah okay, looks like I didn't pay attention there. Anyhow, whatever the issue was, it seems fixed now.
